### ISSUE TITLE
Increase Sleep Timer

### ISSUE THREAD

**Issue by tv0ll** [stuartlangridge/ambient-ubuntu-phone#2]
My wife and I have a baby we are trying to keep asleep with ambient noise and we don't need a sleep timer (or at least need it to go up to around 8 hours). I do not see a way to do something like this currently. We are stuck with the least of 15 minutes and the most of 3 hours.

**Comment by stuartlangridge** [comment]
I'm afraid I don't really have a setup to make an alteration to this app and publish a new version any more. If you're a programmer then you may be able to, though (the relevant part is [here](https://github.com/stuartlangridge/ambient-ubuntu-phone/blob/cbd9222a2276b2e45afbd60fb6424c1af8e68b32/app/Main.qml#L117)).

**Comment by tv0ll** [comment]
What does that value represent? Is your max of 12 equal to 3 hours?

**Comment by stuartlangridge** [comment]
Yes: that's in quarter-hour increments, so a setting of 12 is equal to 3 hours.
